src/mono/sample/mbr/DeltaHelper/DeltaHelper.csproj

```diff
@@ -1,5 +1,5 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
why? considering we're moving towards 7 this seems odd

Reply:
Thanks, that was a mistake.